sample_files/.travis.yml

```diff
@@ -5,9 +5,12 @@ python:
   - "2.7"
 
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  global:
+  - VERSION="8.0"
+  matrix:
+  - LINT_CHECK="1" TESTS="0"
+  - TESTS="1" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
```

Comment:
I wonder if variable like LINT_CHECK and TEST could go in global setted to "0" to be overwritten